instrumentation/netty/netty-4.1/netty-4.1.gradle

```diff
@@ -27,7 +27,7 @@ muzzle {
 }
 
 dependencies {
-  library group: 'io.netty', name: 'netty-codec-http', version: '4.1.0.Final'
+  library group: 'io.netty', name: 'netty-codec-http', version: '4.1.52.Final'
```

Comment:
Wouldn't it be enough to have later versions in `latestDepTest`? I think we still want to test with the oldest supported version.

Reply:
Yep, thanks for that.  Confirmed that with `-PtestLatestDeps=true` and the `addTask` instrumentation commented out the test fails, so reverting the gradle file.